sklearn/compose/tests/test_column_transformer.py

```diff
@@ -120,15 +135,75 @@ def test_column_transformer():
                      transformer_weights['trans2'] * X_res_second1D]).T
     assert_array_equal(both.fit_transform(X_array), res)
     assert_array_equal(both.fit(X_array).transform(X_array), res)
+    assert_allclose(both.inverse_transform(res), X_array)
     assert len(both.transformers_) == 2
 
     both = ColumnTransformer([('trans', Trans(), [0, 1])],
                              transformer_weights={'trans': .1})
     assert_array_equal(both.fit_transform(X_array), 0.1 * X_res_both)
     assert_array_equal(both.fit(X_array).transform(X_array), 0.1 * X_res_both)
+    assert_allclose(both.inverse_transform(0.1 * X_res_both), X_array)
     assert len(both.transformers_) == 1
 
 
+def test_column_transformer_inverse_transform_all_transformers_drop():
```

Comment:
I'm confused about how this test relates to inverse_transform when it is not called.

Reply:
This test was not complete. This PR was updated to include the `inverse_transform` call along with your `pytest.mark.parameterize` suggestion.